crates/apub/src/objects/community.rs

```diff
@@ -101,9 +101,7 @@ impl Object for ApubCommunity {
       inbox: self.inbox_url.clone().into(),
       outbox: generate_outbox_url(&self.actor_id)?.into(),
       followers: self.followers_url.clone().into(),
-      endpoints: self.shared_inbox_url.clone().map(|s| Endpoints {
-        shared_inbox: s.into(),
-      }),
+      endpoints: None,
```

Comment:
I guess this removes the `endpoints: { shared_inbox: ... } ` field from the apub object. Are you sure this doesn't interrupt federation in some way? E.g. other software might have special / performant cases if shared_inbox is present and not detect the main inbox url being the shared inbox

Reply:
I dont know how other software handles this, but the standard doesnt mention anything about inboxes being unique so they should be deduplicated anyway. And deduplication is really the only optimization here.

https://www.w3.org/TR/activitypub/#inbox
https://www.w3.org/TR/activitypub/#shared-inbox-delivery

Reply:
I can't find exact information but it still feels a bit wrong maybe. It might make sense to ask some mastodon people. Regardless I think it would make sense to at least keep the endpoints: {shared_inbox: x} in there even if it's a duplicate of the inbox url.

* https://github.com/w3c/activitypub/issues/242 Is a discussion where the concept of shared inbox was first introduced. Doesn't really help much though.
* The AP spec says "Actor objects MUST have: [...] inbox: A reference to an OrderedCollection comprised of all the messages received by the actor; see [5.2 Inbox](https://www.w3.org/TR/activitypub/#inbox).". This seems to imply that the actor inbox must only contain the messages specifically for this actor. Not sure what that means though.